ehealthid-cli/src/main/java/com/oviva/ehealthid/cli/KeyGeneratorCommand.java

```diff
@@ -35,7 +35,7 @@ public class KeyGeneratorCommand implements Callable<Integer> {
   @CommandLine.Option(
       names = {"-i", "--iss", "--issuer-uri"},
       description = "the issuer uri of the 'Fachdienst' identiy provider",
-      required = true)
+      required = false)
```

Comment:
Why is this optional? Pretty sure that's required for the self-signed certificate. No?
Otherwise we'd at least need to skip cert generation?

Reply:
Because I experimented with extracting it. but that did not yet pan out